peachjam/templates/peachjam/_labels.html

```diff
@@ -0,0 +1,8 @@
+<div class="d-flex align-items-center">
+  {% if labels %}
```

Comment:
Move the `if` outside the `div` -- otherwise there's no point to the if block. We don't want to render an empty div, though.